src/main/kotlin/dev/supergrecko/kllvm/core/Context.kt

```diff
@@ -20,6 +20,14 @@ import org.bytedeco.llvm.global.LLVM
  * @throws IllegalArgumentException If any argument assertions fail. Most noticeably functions which involve a context ref.
  */
 public class Context internal constructor(private val llvmCtx: LLVMContextRef) : AutoCloseable {
+    /**
+     * Control whether the instance has been dropped or not.
+     *
+     * Attempting to do anything with a dead module will fail.
+     */
+    public var isAlive: Boolean = true
+        internal set
```

Comment:
It appears that attempting to do dispose assertions via `Pointer.isNull` would fail on a Context instance which had been dropped. This property was added to keep track of disposed context.